lib/rspecq/worker.rb

```diff
@@ -156,20 +162,24 @@ def reset_rspec_state!
     # Their errors will be reported in the normal flow, when they're picked up
     # as jobs by a worker.
     def files_to_example_ids(files)
-      # TODO: do this programatically
-      cmd = "DISABLE_SPRING=1 bundle exec rspec --dry-run --format json #{files.join(' ')}"
+      cmd = "DISABLE_SPRING=1 bundle exec rspec --dry-run --format json #{files.join(' ')} 2>&1"
       out = `#{cmd}`
 
       if !$?.success?
-        # TODO: emit warning to Sentry
-        puts "WARNING: Error splitting slow files; falling back to regular scheduling:"
-
-        begin
-          pp JSON.parse(out)
+        rspec_output = begin
+          JSON.parse(out)
         rescue JSON::ParserError
-          puts out
+          out
         end
-        puts
+
+        log_event(
+          "Error splitting slow files! Falling back to regular scheduling",
```

Comment:
Maybe an alternative (since we already marked the level as error):

```
Failed to split slow files, falling back to regular scheduling
```

Reply:
Done!